src/LabLog/Migrations/20170928082432_InitialCreate.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace LabLog.Migrations
+{
+    public partial class InitialCreate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "LabEvents",
+                columns: table => new
+                {
+                    Version = table.Column<int>(type: "INTEGER", nullable: false)
+                        .Annotation("Sqlite:Autoincrement", true),
+                    EventBody = table.Column<string>(type: "TEXT", nullable: true),
+                    EventType = table.Column<string>(type: "TEXT", nullable: true),
+                    RoomId = table.Column<Guid>(type: "BLOB", nullable: false),
+                    Timestamp = table.Column<DateTimeOffset>(type: "TEXT", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_LabEvents", x => x.Version);
```

Comment:
primary key needs to be compound of RoomId and Version. If it's just version then two rooms will conflict on versions

Reply:
Automatically generated.